src/components/tree-item/ch-tree-item.tsx

```diff
@@ -147,6 +153,14 @@ export class ChTreeItem {
       this.isLeaf = false;
       this.opened = false;
     }
+
+    //CONFIGURATIONS THAT COME FROM FROM MASTER TREE
+    if (treeRef.checkbox) {
+      this.checkbox = true;
+    }
+    if (treeRef.checked) {
```

Comment:
Remove duplicate if statement

Reply:
They are for different purposes. The first one is for "checkbox", and the second one is for "checked". The "checked" if could be inside "checkbox" if. What do you think? 

```   
if (treeRef.checkbox) {
      this.checkbox = true;
      if (treeRef.checked) {
        this.checked = true;
      }
}
```

Reply:
Okey. In that case, I would do the following:

```
if (treeRef.checkbox) {
  this.checkbox = true;
  this.checked = treeRef.checked;
}
```

Reply:
Sweet. Thanks.